fastapi_route_logger_middleware/__init__.py

```diff
@@ -27,7 +34,8 @@ async def dispatch(self, request: Request, call_next: Callable) -> Response:
 
     def _should_route_be_skipped(self, request: Request) -> bool:
         return any(
-            [path for path in self._skip_routes if request.url.path.startswith(path)]
+            [True for path in self._skip_routes if request.url.path.startswith(path)]
+            + [True for regex in self._skip_regexes if regex.match(request.url.path)]
```

Comment:
If somebody has a long list of either one of these, this could be optimized to loop and return on first match, rather than iterate over the entirety of both lists. But I suspect that is an uncommon use case.